omnigibson/prims/entity_prim.py

```diff
@@ -471,6 +418,26 @@ def links(self):
             dict: Dictionary mapping link names (str) to link prims (RigidPrim) owned by this articulation
         """
         return self._links
+    
+    @property
+    def articulation_tree(self):
```

Comment:
is this used for the scene graph?

Reply:
its used for the AG joint type decision (but sure, why not in the scene graph too!)